extensions/amp-a4a/rtc-documentation.md

```diff
@@ -196,6 +196,143 @@ In both cases, the requirements for an errorReportingUrl are the same:
 
 The error ping will be sent by creating an image pixel in the document. See `sendErrorMessage` in [real-time-config-manager.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/real-time-config-manager.js) for implementation details.
 
+### AMP Consent Integration
+
+By default, if the consent state of the page is not `SUFFICIENT` or `UNKNOWN_NOT_REQUIRED`, then no RTC callouts will be sent from the page. However, you may optionally modify this setting, to whitelist specific RTC callouts that should be sent regardless of the consent state. You can modify this across all RTC requests for a given ad slot or on a per-RTC-callout basis. You also may either whitelist for all consent states, or only specific consent states, using the RTC Config attribute `sendRegardlessOfConsentState`. 
```

Comment:
Page is a bit of a misnomer as at some point I believe amp-consent may allow multiple consent templates.  Let's start this section by indicating how amp-consent integrates (there is pre-existing documentation) and re-word this to something like:  If amp-consent response is neither SUFFICIENT nor UNKNOWN_NOT_REQUIRED, then by default all RTC call-outs are suppressed.  However, this can be overridden by...

Reply:
done